x/model/types/errors.go

```diff
@@ -124,3 +125,9 @@ func NewErrLsfRevisionIsNotValid(previousLsfVersion interface{},
 		"LsfRevision %v should be greater then existing lsfRevision %v by 1",
 		currentLsfVersion, previousLsfVersion)
 }
+
+func NewErrModelDeletionCertified(vid interface{}, pid interface{}, softwareVersion interface{}) error {
```

Comment:
We also need a second method NewErrModelVersionDeletionCertified with error message
`Model version associated with vid=%v, pid=%v and softwareVersion=%v has a compliance record and can't be deleted`